### PR TITLE
Navigation modal

### PR DESCRIPTION
**Pull request by dipen2000**
I created this navigation bar and modal so please review my PR and give me feedback.

### CODE REVIEW

File: CSS/documentation.css [hunk 1]
```diff
@@ -52,11 +53,11 @@ nav{
 /********************** content section of the page  ******************/
 
 main{
-    margin-top : 30px;
```
Comment (j24m) [inline CSS/documentation.css:55]: rem?

File: CSS/documentation.css [hunk 2]
```diff
@@ -52,11 +53,11 @@ nav{
...
-    border : 3px solid red;
+    margin-top : 3rem;
+    /* border : 3px solid red; */
```
Comment (j24m) [inline CSS/documentation.css:57]: remove commented code everywhere in the file before merging.

**Review by j24m — APPROVED** [review]
I saw your code, you write really clean code and understandable class names, I haven't added comment on every line so telling here to make a few changes before merging, 
1.Remove commented code.
2.Use rems for more than 10px and px for less than  1 rem.

File: components/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.navigation-bar{
+    width : 100rem;
```
Comment (ShrutiLakhani) [inline components/navigation/navigation.css:2]: May be you wanna use % here.

File: components/navigation/navigation.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.lib-name a{
+    font-family: var(--library-logo-font);
+    font-size : 3.5rem;
+    /* color : rebeccapurple; */
```
Comment (ShrutiLakhani) [inline components/navigation/navigation.css:8]: Please delete commented code

File: components/modal/modal.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    line-height : 1.3;
+    margin : 0.8rem 0;
+}
+
+.modal-foot{
+    margin: 0.5rem 0;
+    text-align: right;
+    padding: 0.5rem;
+}
+
```
Comment (ShrutiLakhani) [inline components/modal/modal.css:27]: Clean code. Looks good to merge

**Review by ShrutiLakhani — APPROVED** [review]
Your code is clean. Pls make suggested changes and its good to merge. Great job

**Pull request merged**